resources/views/install/config.blade.php

```diff
@@ -0,0 +1,748 @@
+                  <div class="d-flex flex-column flex-fill flex-shrink-0 p-3 text-bg-dark">
+                    <div class="d-flex align-items-center mb-3 mb-md-0 me-md-auto text-white text-decoration-none">
+                        <div class="text-center">
+                            <img class="" src="/img/badgerstaffpanel-logo.png" width="64" height="64" alt="website-logo">
+                            <h5>Installation Script</h5>
+                        </div>
+                    </div>
+                    <hr>
+                    <ul class="nav nav-pills flex-column"> 
+                      <li class="nav-item"><a href="welcome" class="nav-link text-white"><i class="bi text-success bi-check2-square me-2"></i> Welcome</a></li>
+                      <li class="nav-item"><a href="welcome" class="nav-link text-white"><i class="bi text-success bi-check2-square me-2"></i> Agreements</a></li>
+                      <li class="nav-item"><a href="{{ url()->current()}}" class="nav-link active" aria-current="page"><i class="bi bi-square me-2"></i> Configuration</a></li>
+                      <li class="nav-item"><a href="{{ url()->current()}}" class="nav-link text-white"><i class="bi bi-square me-2"></i> Database</a></li>
+                      <li class="nav-item"><a href="{{ url()->current()}}" class="nav-link text-white"><i class="bi bi-square me-2"></i> Administrative User</a></li>
+                      <li class="nav-item"><a href="{{ url()->current()}}" class="nav-link text-white"><i class="bi bi-square me-2"></i> Discord</a></li>
+                      <li class="nav-item"><a href="{{ url()->current()}}" class="nav-link text-white"><i class="bi bi-square me-2"></i> Groups</a></li>
+                      <li class="nav-item"><a href="{{ url()->current()}}" class="nav-link text-white"><i class="bi bi-square me-2"></i> Complete</a></li>
+                    </ul>
+                    <hr>
+                  </div>
+                </div>
+            </div>
+            <div class="col-sm m-5 align-items-start">
+              <form id="env_creation_tool" class="row gy-2">
+                @csrf
+                <div class="col-12">
+                    <h4>Configure your install</h4>
+                    <div>
+                        Now we get to making important changes to your installation making it functional. Below you will see a lot of settings to configure. In some places there pre-filled options we made as recommendations based
+                        on the decisions you make in the first section, and based on automatically detected system parameters. If you are unsure about what to set for an option it may be best left to the recommended options, or 
+                        should you have trouble with one not pre-filled, you may refer to our documentation or support discord. 
+                    </div>
+                </div>
+                <div id="error-showcase" class="col-12 my-2" hidden>
```

Comment:
If this will be used in several files, you should probably make a partial blade file for it